src/lib/providers/BidProvider.tsx

```diff
@@ -186,9 +189,10 @@ const BidProvider: React.FC<BidProviderType> = ({ children }) => {
 			)) as zAuction.BidDto[];
 
 			try {
-				bids = bids.filter((e) => {
-					return e.account.toLowerCase() !== domain.owner.id.toLowerCase();
-				});
+				if (filterOwnBids)
```

Comment:
Minor nitpick;
Please use brackets on if statements, even if it's one statement;

ie:
```
if (filterOwnBids) {
... code
}
```